packages/cerebral/package.json

```diff
@@ -35,8 +35,5 @@
       "**/*.test.js",
       "**/testHelper.js"
     ]
-  },
-  "devDependencies": {
-    "mock-socket": "^6.0.4"
```

Comment:
`mock-socket` does not seem to be used in cerebral testing so this is good cleanup.